src/hyperfine/export/mod.rs

```diff
@@ -55,7 +55,9 @@ impl ExportManager {
     }
 
     /// Add an additional exporter to the ExportManager
-    pub fn add_exporter(&mut self, export_type: ExportType, filename: &str) {
+    pub fn add_exporter(&mut self, export_type: ExportType, filename: &str) -> Result<()> {
+        let _ = File::create(filename)?;
```

Comment:
Another idea could be to open the file here (in write-mode, as below) and store the file handle with the exporter - instead of (or in addition to) the filename.

Reply:
Good idea. If we add `file: File` to the struct, we however need to use the ExportManager as `&mut` everywhere since we need a `&mut self` in `write_results`. It works, but requires more changes. Did you think of a different way?

Reply:
Let's leave it as it is for now - thank you!